### PR TITLE
Now the menu pops by hovering "?" is completely under display

### LINKED ISSUE #2671: menu under "?" is not completely inside the screen
**Actual Behaviour**
menu that pops by hovering on "?" is not completely inside the screen 
susi1
**Expected Behaviour**
menu should be completely inside the the screen

**Would you like to work on the issue?**
Yes , i would like to work on it

### PR DESCRIPTION
**Pull request by tanujvyas10**
Fixes #2671

**Changes: Now the menu that pops on hovering on "?" is completely inside the display console**
**Demo link**:
[https://steadfast-babies.surge.sh/](https://steadfast-babies.surge.sh/)

### CODE REVIEW

**Comment by tanujvyas10** [comment]
@fragm3  and @samagragupta  thank you sir for your advise , i have re-created the pull request,Please review my work.

File: package.json [hunk 1]
```diff
@@ -100,11 +101,11 @@
     "gh-pages": "2.0.1",
     "husky": "2.4.0",
     "lint-staged": "8.2.0",
+    "material-table": "^1.39.2",
     "prettier": "1.18.1",
     "react-scripts": "3.0.1",
     "react-test-renderer": "16.8.6",
-    "redux-devtools-extension": "^2.13.8",
-    "material-table": "^1.39.2"
```
Comment (samagragupta) [inline package.json]: Why did you remove that?

Comment (tanujvyas10) [reply package.json]: i didnot remove them by cause,i was setting up the project on my local server , may at that time some dev dependencies got remove. i will add them again

**Comment by tanujvyas10** [comment]
> @tanujvyas10 do not add yarn.lock file, and remove all unnecessary changes in package.json done


Sir, they are not changes, they are just rearranged dev-dependencies ,when i setup the proj on my local server.
